app/Http/Controllers/Api/TagsController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Rogue\Http\Controllers\Api;
+
+use Rogue\Http\Requests\TagsRequest;
+use Rogue\Repositories\PostRepository;
+use Rogue\Http\Transformers\PostTransformer;
+
+class TagsController extends ApiController
+{
+    /**
+     * The post service instance.
+     *
+     * @var Rogue\Repositories\PostRepository
+     */
+    protected $post;
+
+    /**
+     * @var \League\Fractal\TransformerAbstract;
+     */
+    protected $transformer;
+
+    /**
+     * Create a controller instance.
+     *
+     * @param  PostContract $posts
+     * @return void
+     */
+    public function __construct(PostRepository $post)
+    {
+        $this->post = $post;
+        $this->transformer = new PostTransformer;
+    }
+
+    public function store(TagsRequest $request)
```

Comment:
Sure, but right now they are exactly the same. I can't imagine us really wanting the functionality to be different especially for tagging, and the only difference is the way we authorize the request.